### ISSUE TITLE
About stage1 training steps 

### ISSUE THREAD

**Issue by Vincent-luo** [Zejun-Yang/AniPortrait#182]
I find in the "stage1.yaml" file `max_train_steps` is set to 300,000, and I want to ask is it really necessary to train such a long schedule? In animate anyone paper, they say just train stage1 for 30,000 steps. And in my own experiment the loss doesn't change too much after 30,000 steps. It would be helpful if you can provide some guidance about how to find a proper training schedule.
